influxql/query_executor.go

```diff
@@ -66,6 +66,9 @@ type ExecutionOptions struct {
 
 	// Node to execute on.
 	NodeID uint64
+
+	// Do not log the normalized statement for this query.
+	Quiet bool
```

Comment:
I'm not too sure I like that the idea of having this be a per-query option. I would rather see this as a switch of "log queries" or "don't log queries".